types/signed_header_test.go

```diff
@@ -1,27 +1,39 @@
 package types
 
 import (
-	"strconv"
+	"fmt"
 	"testing"
 	"time"
 
 	"github.com/celestiaorg/go-header"
+	"github.com/cometbft/cometbft/crypto/ed25519"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestVerify(t *testing.T) {
+func TestSignedHeader(t *testing.T) {
+	// Generate a random signed header
 	trusted, privKey, err := GetRandomSignedHeader()
 	require.NoError(t, err)
+
+	// Wait for a second to ensure time difference before generating the next header
 	time.Sleep(time.Second)
+
+	// Get the next random header
 	untrustedAdj, err := GetNextRandomHeader(trusted, privKey)
```

Comment:
GetNextRandomHeader should just set the time to plus 1 second so that the tests don't need to wait a second.